driver/bpf/fillers.h

```diff
@@ -3518,6 +3518,12 @@ FILLER(sys_pagefault_e, false)
 	return res;
 }
 
+#ifdef SEND_SIG_FORCED
+#define SIGINFO_NOT_A_POINTER(_info) ((_info) <= SEND_SIG_FORCED)
+#else
+#define SIGINFO_NOT_A_POINTER(_info) ((struct kernel_siginfo*)(_info) <= SEND_SIG_PRIV)
```

Comment:
Why cast for this one and not the other #define? We should use a function and get a little bit of type-checking safety.

Reply:
Unfortunately, [this change](https://github.com/torvalds/linux/commit/ae7795bc6187a15ec51cf258abae656a625f9980) renamed the struct type. Since the changes happened around the same time (21 days apart) I got lazy and decided to use the presence of SEND_SIG_FORCED as an indicator for whether the symbols would be the old type or the new type.

(The two types are byte-compatible, the kernel_ one is just larger).

I'll go ahead and change from a macro to a function though.